editor/js/editor-libs/codemirror-editor.js

```diff
@@ -3,19 +3,18 @@ import { EditorState } from "@codemirror/state";
 import * as commands from "@codemirror/commands";
```

Comment:
There's a lot of changes going on in this file that don't seem to be related to ES6 conversion, including formatting changes that don't match MDN's style.  Can we revert this file and defer any changes to another PR?